cli/commands/genesis/root.go

```diff
@@ -24,25 +24,26 @@ import (
 	"github.com/berachain/beacon-kit/chain"
 	"github.com/berachain/beacon-kit/consensus-types/types"
 	"github.com/berachain/beacon-kit/errors"
-	"github.com/berachain/beacon-kit/primitives/bytes"
+	"github.com/berachain/beacon-kit/primitives/common"
 	"github.com/berachain/beacon-kit/primitives/encoding/json"
 	"github.com/berachain/beacon-kit/primitives/math"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
+// Beacon, AppState and Genesis are code duplications that
+// collectively reproduce part of genesis file structure
+
+type Beacon struct {
```

Comment:
minor cleanup of these types to unlock UTs.
We really should not need this. We should have a single type representing the genesis file structure and reuse it anywhere.
Stuff for another PR